packages/components/radio/src/radio.styles.ts

```diff
@@ -0,0 +1,90 @@
+import styled from '@emotion/styled';
+
+interface RadioStyledProps {
+  backgroundColor?: string;
+}
+
+export const LabelStyled = styled.label`
+  margin-left: 8px;
+  cursor: pointer;
+`;
+
+export const InputStyled = styled.input<{ backgroundColor?: string }>`
+  appearance: none;
+  width: 14px;
+  height: 14px;
+  display: grid;
+  place-content: center;
+  border-radius: 50%;
+  border: 1px solid #767676;
+  transition: background-color 0.3s ease-in-out;
+
+  &::before {
+    content: "";
+    width: 6px;
+    height: 6px;
+    border-radius: 50%;
+    transform: scale(0);
+    transition: 120ms transform ease-in-out;
+    background-color: #298DCC;
+  }
+
+  &:checked {
+    background-color: #FFFFFF;
+    border-color: #298DCC;
+
+    &::before {
+      transform: scale(1);
+    }
+
+    &:hover {
+      border-color: #005E99;
+
+      &::before {
+        background-color: #005E99;
+      }
+    }
+
+    &:active {
+      border-color: #003A5E;
+
+      &::before {
+        background-color: #003A5E;
+      }
+    }
+
+    &:disabled {
+      border-color: #BCBCBC;
+
+      &::before {
+        background-color: #BCBCBC;
+      }
+    }
+  }
+
+  &:hover {
+    border-color: #2D2D2D;
+  }
+
+  &:disabled {
+    border-color: #BCBCBC;
+    cursor: not-allowed;
+
+    + label {
+      color: #BCBCBC;
+      pointer-events: none;
+    }
+  }
+`;
+
+export const RadioStyled = styled.div<Omit<RadioStyledProps, 'label'>>`
+  display: flex;
+  align-items: center;
+  background-color: ${props => props.backgroundColor || 'transparent'};
```

Comment:
When using a double pipe after a string value, it will be unreachable because a double pipe expects a boolean first. 
Nullish Coalescing expects a null or undefined to return the fallback value.

```suggestion
  background-color: ${props => props.backgroundColor ?? 'transparent'};
```